packages/pilot/src/components/Event/index.js

```diff
@@ -3,21 +3,35 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
```

Comment:
Como na sugestão do @Deivis, troca tudo desse `cx` para `classNames`.

```suggestion
import classNames from 'classnames'
```